### ISSUE TITLE
Beta Testing

### ISSUE THREAD

**Issue by jbonda** [TAWinterton2/CS-F600-ML-GUI#73]
1.  While uploading a Zip file with different file formats, it generates an error in terms of different column numbers and different column names.  Its good and more informative if the error message could be more specific.

Concerns: 1. Does it mean this website will not accept a zipped file if it has file formats rather than CSV or Is there an option to choose The CSV among different formatted files from the Uploaded CSV file?

2. Displaying a warning if the test size of data is more than train size. It is also good to provide the same warning in case Test size data is equal to train size data.Because, Its always better to have low test size data for an optimal ML model.

3. Label the model training graph.

**Comment by raelewis** [comment]
Thanks for catching these!
1. If the user tries to submit an archive that contains any files besides .csv files, it will be rejected. If the files within an archive have differing structures (column names, column numbers), the archive is rejected. The front end also has a comment on this as well for the users so they know what is expected in an archive prior to submission.
2. We've updated our error checking to throw an error if test size equals train size. 
3. We've added labels to the model training graph, as well as some additional output adjustments to maintain uniformity among displayed graphs.
